fcrepo-http-api/src/test/java/org/fcrepo/integration/http/api/FedoraLdpIT.java

```diff
@@ -2001,29 +1988,6 @@ public void testIngestOnSubtree() throws IOException {
         assertEquals(serverAddress + id + "/x", getLocation(method));
     }
 
-    /**
-     * Ensure that the objects cannot be pairtree child resources
-     *
-     * @throws IOException in case of IOException
-     */
-    @Test
-@Ignore
-    public void testIngestOnPairtree() throws IOException {
```

Comment:
Pairtrees are no longer a thing

Reply:
Should this test be re-tooled for "ghost nodes"?

Reply:
We have a test of GhostNodes here (https://github.com/fcrepo/fcrepo/blob/main/fcrepo-http-api/src/test/java/org/fcrepo/integration/http/api/FedoraLdpIT.java#L4946) which I think covers this.

Reply:
Actually that test tries to create at the ghost node. I'll add an attempt to just POST to the ghost node.